packages/playwright-core/src/server/injected/roleUtils.ts

```diff
@@ -461,6 +461,38 @@ export function getElementAccessibleDescription(element: Element, includeHidden:
   return accessibleDescription;
 }
 
+export function getElementAccessibleErrorMessage(element: Element, includeHidden: boolean): string {
```

Comment:
It seems like `includeHidden: true` is never passed, so we might as well not support it.

Reply:
The `includeHidden` parameter has been removed. I appreciate your feedback! :)